notebook/notebook/notebook.py

```diff
@@ -94,14 +123,15 @@ def start_pod(jupyter_token, image):
                 command=[
                     'jupyter',
                     'notebook',
+                    "--ip", "0.0.0.0", "--no-browser",
                     f'--NotebookApp.token={jupyter_token}',
                     f'--NotebookApp.base_url=/instance/{svc.metadata.name}/'
                 ],
                 name='default',
                 image=image,
                 ports=[kube.client.V1ContainerPort(container_port=8888)],
                 resources=kube.client.V1ResourceRequirements(
-                    requests={'cpu': '1.601', 'memory': '1.601G'}),
+                     requests={'cpu': '1.601', 'memory': '1.601G'}),
```

Comment:
Generally prefer mod 4 spaces of indentation.

Reply:
mod? I'm happy to use whatever indenting rules you like.